cfy_manager/components/usage_collector/scripts/collect_cloudify_usage.py

```diff
@@ -8,9 +8,13 @@
 from manager_rest.storage import models
 from script_utils import (logger,
```

Comment:
no idea how does this pass py3 flake/py3_compat, the implicit relative import. Should most likely be something like `from .script_utils import ...`

Reply:
but if it works, dont worry for now

Reply:
Great point, I've read bout it too and tried to change it to an explicit import but it failed since during the [installation of the usage_collector this file is copied to a different directory](https://github.com/cloudify-cosmo/cloudify-manager-install/blob/master/cfy_manager/components/usage_collector/usage_collector.py#L96) and then the explicit declaration is not working. Might be something I need to solve now?